webapp/src/js/services/z-score.js

```diff
@@ -70,15 +70,10 @@ angular.module('inboxServices').factory('ZScore',
     };
 
     var init = function() {
-      return DB().get(CONFIGURATION_DOC_ID)
-        .then(function(doc) {
-          tables = doc.charts;
-        })
-        .catch(function(err) {
-          if (err.status === 404) {
-            return;
-          }
-          throw err;
+      // use allDocs instead of get so a 404 doesn't report an error
+      return DB().allDocs({ key: CONFIGURATION_DOC_ID, include_docs: true })
+        .then(function(result) {
+          tables = result.rows.length && result.rows[0].doc.charts;
```

Comment:
Will `result.rows[0].doc.charts` throw an error in the case of a 404? I would expect result to look like:
```
{
  rows: [{
    id: 'blah,
    error: true,
    status: 404
  }]
}
```

Reply:
In my testing it returns a 0 length array. I'll double check...

Reply:
Confirmed against couchdb:

Request: `/medic/_all_docs?key="zscore"`
Response:

```json
{
  "total_rows": 657,
  "offset": 656,
  "rows": []
}
```

I'll add a unit test to prove this is handled correctly.